entity-framework/core/modeling/types-and-tables.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Types and Tables - EF Core
+author: roji
+ms.date: 12/03/2019
+ms.assetid: cbe6935e-2679-4b77-8914-a8d772240cf1
+uid: core/modeling/types-and-tables
+---
+# Types and Tables
+
+Including a type on your context means that it is included in EF Core's model; we usually refer to such a type as an *entity*. EF Core can read and write entity instances from/to the database, and if you're using a relational database, EF Core can create tables for your entities via migrations.
```

Comment:
First sentence is ambiguous. Are you referring to DbSet?

Reply:
Changing to "Including a DbSet of a type", tell me what you think.